include/matx/core/type_utils.h

```diff
@@ -126,7 +126,7 @@ struct is_matx_op_impl<T, std::void_t<typename T::matxop>> : std::true_type {
  * 
  * @tparam T Type to test
  */
-template <typename T> constexpr bool is_matx_op()
+template <typename T> constexpr __MATX_HOST__ __MATX_DEVICE__ bool is_matx_op()
```

Comment:
Did you mean to keep the device qualifier here?

Reply:
Same as others